posts/flexible-data-model-design.md

```diff
@@ -0,0 +1,349 @@
+---
+authors:
+- Pasha Paterson
+tags:
```

Comment:
I guess you're the new blog czar so you could decide what tags are good tags. Only `Java` is an existing tag in the system. Also, I couldn't find any perfect synonym replacements for the others.

Reply:
image is missing for the jumbo

Reply:
Do tags have to be created manually in the blogging system if no one has ever used them before? That seems like it would be... less than ideal. 😕 

I am new to the Ippon blog scene. Should a banner image be added as part of the blog entry, where we store the images in the other directory? Do we typically source them from a particular place? Hmm. 🤔

Reply:
I think new tags are added automatically. Alexis removed a bunch recently because they were underutilized. I personally think these last three tags are pretty relevant and general enough.

Reply:
https://github.com/ippontech/blog-usa/blob/master/README.md#metadata
According to this, non-existant tags are ignored.

Reply:
Oh great, that's helpful, thanks Ghost. 😅